app/assets/stylesheets/components/_buttons.scss

```diff
@@ -26,7 +26,19 @@
   &:hover {
     background-color: darken($red, 6%);
   }
-}  
+}
+
```

Comment:
Alicia suggested I use a different blue for the "View Loss Report" button so I added this button style, let me know if I should do this differently